### ISSUE TITLE
Domain-centric lightning bolt

### ISSUE THREAD

**Issue by mor10** [MammothHQ/Known-Issues#6]
This is directly related to how Google handles their content. Try pinning the Inactive Account Manager in this story to a board: http://dataliberation.blogspot.ca/2013/04/plan-your-digital-afterlife-with.html

Because the link is to a google.com entity, every Google.com page opened (calendar, Gmail, Drive etc) will be flagged as pinned with this link. 

I have no smart solution for this as it has to do more with how Google handles everything under one domain, but I can imagine this happening across Apple and Microsoft entities as well.

**Comment by karanmg** [comment]
I totally agree with this. Delivering a fix for this has been on the backburner for a while, since it will be resource intensive. 

Possible solution: 
- indicate partial/exact domain match in "reminders" list
- different lightning bolt based on exact/partial domain match
